server/test/services/ti/TrustedIntermediaryServiceTest.java

```diff
@@ -271,7 +271,7 @@ public void addClient_WithEmptyEmailAddress() {
             .filter(acct -> acct.getApplicantName().equals("Email, No"))
             .findFirst()
             .get();
-    assertThat(account.getApplicants().get(0).getApplicantData().getDateOfBirth().get().toString())
+    assertThat(account.getApplicants().get(0).getDateOfBirth().get().toString())
```

Comment:
suggestion (non-blocking): same comment as above, keep tests for both code paths.